pkgs/os-specific/darwin/sketchybar/default.nix

```diff
@@ -34,7 +35,7 @@ stdenv.mkDerivation rec {
     description = "A highly customizable macOS status bar replacement";
     homepage = "https://github.com/FelixKratz/SketchyBar";
     platforms = platforms.darwin;
-    maintainers = [ maintainers.azuwis ];
+    maintainers = with maintainers; [ azuwis khaneliman ];
```

Comment:
Please split this change into a separate commit, `sketchybar: add khaneliman as maintainer`.